app/WikiSetting.php

```diff
@@ -36,6 +36,8 @@ class WikiSetting extends Model
     public const wgSecretKey = 'wgSecretKey';
     public const wgLogo = 'wgLogo';
     public const wgFavicon = 'wgFavicon';
+    public const wgOAuth2PrivateKey = 'wgOAuth2PrivateKey';
+    public const wgOAuth2PublicKey = '$wgOAuth2PublicKey';
```

Comment:
```suggestion
    public const wgOAuth2PublicKey = 'wgOAuth2PublicKey';
```

suggestion to make naming scheme the same as the other constants (without $)